azavea/azavea-branding-guide
# Add minimal development environment setup

## Overview

This was necessary to test the state of `develop` before merging it into `master`. It uses Docker and Docker Compose to run Jekyll inside of a Ruby 2.5 environment with relevant dependencies pinned.

It also includes a `livereload` setup for the pushing of local changes to the browser.

## Testing Instructions

Execute the following command and navigate to http://localhost:4000. Note that this requires a functional Docker based development environment.

```bash
$ docker-compose up
```

Closes #50
